testsuite/shared/enable-elytron-domain.cli

```diff
@@ -1,36 +1,40 @@
 embed-host-controller
 
+/profile=default/subsystem=elytron/http-authentication-factory=ejb-http-authentication:add(security-domain=ApplicationDomain, http-server-mechanism-factory=global,mechanism-configurations=[{mechanism-name=BASIC}])
 /profile=default/subsystem=undertow/application-security-domain=other:add(security-domain=ApplicationDomain)
 /profile=default/subsystem=undertow/server=default-server/host=default-host/setting=http-invoker:undefine-attribute(name=security-realm)
-/profile=default/subsystem=undertow/server=default-server/host=default-host/setting=http-invoker:write-attribute(name=http-authentication-factory, value=application-http-authentication)
+/profile=default/subsystem=undertow/server=default-server/host=default-host/setting=http-invoker:write-attribute(name=http-authentication-factory, value=ejb-http-authentication)
```

Comment:
Is this space for further simplification? I mean can be same arguments why this PR was created applied to EJB over Http?